README.md

```diff
@@ -33,8 +29,45 @@ to UUID generation, potentially using `Math.random()`.
 
 It's well documented that
 [`Math.random()` is not cryptographically secure](https://v8.dev/blog/math-random), by instead
-exposing users to the standard library UUID module we prevent the pitfalls that go hand in hand
-with home-grown implementations.
+exposing users to the standard library UUID we prevent the pitfalls that go hand in hand with
+home-grown implementations.
+
+## Overview
+
+The `uuid` built-in provides an API for generating RFC 4122 identifiers.
+
+The default export of the `uuid` module is the
+[Version 4 Algorithm](https://tools.ietf.org/html/rfc4122#section-4.4), and returns the string
+representation _(as described in RFC-4122)_.
+
+```js
+import uuid from "lib:uuid"; // (Note: exact import syntax is TBD).
+uuid(); // 52e6953d-edbe-4953-be2e-65ed3836b2f0
+```
+
+## Out of scope
+
+Algorithms described in RFC 4122 other than Version 4 are not initially supported.
+
+Statistics we've collected
+([see issue #4](https://github.com/bcoe/proposal-standard-module-uuid/issues/4)) indicate that the
+Version 4 algorithm is most widely used:
+
+| Algorithm Version | Repo Count | %    |
+| ----------------- | ---------- | ---- |
+| v4                | 18318      | 79.7 |
+| v1                | 4399       | 19.1 |
+| v5                | 231        | 1    |
+| v3                | 29         | .1   |
+
+### Reasons for not Supporting Other UUID Versions
+
+While there is utility in other UUID versions, we are advocating starting with the minimal API
+surface possible, that supports a large percentage of users.
+
+It is expected that additional functionality such as version 1, 3, and 5 UUIDs, parsing and
+validation, and binary uuid formats will be provided in future versions of this specification, and
```

Comment:
This is expressing a lot of confidence that there will be such a follow-on proposal. Regardless of how strong evidence we can develop that it's not needed, I think we should say here that it *could* be provided *if* we later determine that it's motivated, and that we have not determined yet that it's motivated.

Reply:
Fair point.  The goal of this statement is simply to reassure readers that nothing in this version of the API will interfere with addressing future needs.